### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by betty5257**

### CODE REVIEW

**Comment by yatharthranjan** [comment]
Can the README please also include an example of what data is stored in the csv file? Also is it possible for the cordova application that is using this plugin to have access to the OpenSmile config file ?
Thanks.

File: plugin.xml [hunk 1]
```diff
@@ -22,10 +23,10 @@
         <param name="android-package" value="myjni.OpenSmile"/>
       </feature>
     </config-file>
-	<config-file target="AndroidManifest.xml" parent="/manifest">
+    <config-file target="AndroidManifest.xml" parent="/manifest">
         <uses-permission android:name="android.permission.INTERNET"/>
```
Comment (yatharthranjan) [inline plugin.xml:27]: What is the `INTERNET` permission used for?

File: README.md [hunk 2]
```diff
@@ -12,10 +12,8 @@ Create a new Cordova Project
     
 Install the plugin
 
-    $ cd /to/project/folder/
-
-    $ cordova plugin add https://github.com/RADAR-CNS/RADAR-Cordova-OpenSmile-Plugin.git
-
+    $ cd hello
+    $ cordova plugin add https://github.com/nithinth7/plugin-opensmile.git
```
Comment (yatharthranjan) [inline README.md]: Should this be `RADAR-base/RADAR-Cordova-OpenSmile-Plugin` ?

**Comment by betty5257** [comment]
The csv file should include acoustic features extracted from the active
speech recorded in the app. Ideally, it should be the raw audio files
rather than high-level features that should be stored. But there are still
some on-going discussion to address this issue.
It could have access to the opensmile config file.

File: plugin.xml (hunk 1 above)
Comment (betty5257) [reply plugin.xml:27]: The former developers were gone and did not reply to my email regarding these changes. Could you please simply have a trail by testing what happens when this permission is commented out? Thanks a million.

File: README.md (hunk 2 above)
Comment (betty5257) [reply README.md]: you are right. corrected now.

**Comment by yatharthranjan** [comment]
Okay cool. i will add an issue to track this.

**Review by yatharthranjan — APPROVED** [review]
Looks good to me. Thanks

**Comment by yatharthranjan** [comment]
Hi @nickcummins41 @betty5257, is there something this PR is waiting for??
We need to start using this in the Questionnaire app. Or should we use this one ? -- https://www.npmjs.com/package/cordova-plugin-opensmile

**Comment by nickcummins41** [comment]
I don't believe we are waiting on anything

**Comment by yatharthranjan** [comment]
Okay great. Proceeding with the merge then.

**Pull request merged**